backend/src/dao/TeamEventsDao.ts

```diff
@@ -16,10 +16,16 @@ export default class TeamEventsDao {
           numCredits,
           hasHours,
           requests: await Promise.all(
-            requests.map((ref) => ref.get().then((doc) => doc.data() as IdolMember))
+            requests.map((ref) => ({
+              ...ref,
+              member: ref.member.get().then((doc) => doc.data()) as unknown as IdolMember
```

Comment:
is the `as unknown` needed?

Reply:
Without `as unknown`, the following error takes place.

<img width="823" alt="Screen Shot 2022-02-14 at 4 58 50 PM" src="https://user-images.githubusercontent.com/70241445/153953498-e323acee-0163-4f4f-9405-ba0d7693f6d3.png">